### ISSUE TITLE
{Adaptive Cards - Activity Update}: Screen Reader is not conveying the copied information when activating the Copy button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9195]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Win11
OS Version: 24H2 (OS Build 22631.5189)
URL: https://adaptivecards.io/
Edge dev Browser version: 137.0.3255.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**

1. Open the above URL on edge dev browser.
2. Home page will appear.
3. Press tab key and navigate to the Sample tab item and select it.
4. Press tab key and navigate to the 'Activity Update' option and press enter key to activate it.
5. Press tab key and navigate to the 'Copy' button and press enter key to activate it and observe the issue.

**Actual Result:**
Screen Reader is remain silent and not announcing any information when activating the 'Copy sample data json' button.

**Expected Result:**
After activating the 'Copy' button screen reader should announce the information and it should announce as **'Copied.'**

**Note:**
Same issue is repro with NVDA.

**User Impact:**
It will impact screen reader user as they will not be able to know whether the action is taken on the control or not?

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

### Screenshots

https://github.com/user-attachments/assets/02aa17a8-d725-4767-bf24-2928b2aeaf3e

![Image](https://github.com/user-attachments/assets/610d1993-b261-4890-9d4e-8db2582e5c11)

### Card JSON

```JSON
NA
```
